client/extensions/woocommerce/state/action-types.js

```diff
@@ -103,6 +103,31 @@ export const WOOCOMMERCE_SERVICES_SHIPPING_ACTION_LIST_CREATE =
 export const WOOCOMMERCE_SETTINGS_BATCH_REQUEST = 'WOOCOMMERCE_SETTINGS_BATCH_REQUEST';
 export const WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS =
 	'WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS';
+export const WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS = 'WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS';
```

Comment:
Just a note that you are adding a second export of `WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS` here. Saw this when testing the API PR. It causes an error when booting Calypso.

Reply:
Yes, bad rebase. Noticed it also.